client/state/plugins/installed/test/fixtures/plugins.js

```diff
@@ -59,21 +78,22 @@ export const jetpack = {
 		},
 	},
 };
+export const jetpackWithSites = placeSitePropsOnSiteObject( jetpack );
 
 export const jetpackUpdated = {
 	id: 'jetpack/jetpack',
 	slug: 'jetpack',
-	active: true,
 	name: 'Jetpack by WordPress.com',
 	plugin_url: 'http://jetpack.com',
-	version: '4.2.2',
 	description:
 		'Bring the power of the WordPress.com cloud to your self-hosted WordPress. Jetpack enables you to connect your blog to a WordPress.com account to use the powerful features normally only available to WordPress.com users.',
 	author: 'Automattic',
 	author_url: 'http://jetpack.com',
 	network: false,
-	autoupdate: true,
 	log: [ 'Array' ],
+	active: true,
+	version: '4.2.2',
+	autoupdate: true,
```

Comment:
Just curious: why are these changes necessary?

Reply:
The plugin objects which are returned from the server always come from a site-specific endpoint, so they have these site-specific properties on them. However in the store the objects are modified so that site-specific properties are found on a site object in order to accommodate cases where plugins on multiple sites are being displayed.

So I need both versions of these objects for the unit tests. Mocked server responses use the version with the site-specific properties on the object, while calls to actions that would be made from component code use the version with the site-specific properties on site objects.

The rearranging of the props was an artifact of my development process which reduced the readability of the diff, so I undid those changes in https://github.com/Automattic/wp-calypso/pull/76684

Reply:
I might be misunderstanding, but I still see those even though the branch is based on the one from https://github.com/Automattic/wp-calypso/pull/76684. Not a big deal of course.

Noting that the order of object properties has historically been non-guaranteed, so a more reliable way to write the test assertions is to use [`expect.objectContaining`](https://jestjs.io/docs/expect#expectobjectcontainingobject).